src/components/organisms/modals/FormProgressModal/FormProgressModal.tsx

```diff
@@ -0,0 +1,82 @@
+import ModalBase, {
+  ButtonPositionTypes,
+  TitleFontTypes,
+} from 'components/organisms/ModalBase/ModalBase'
+import { AppearanceTypes } from 'components/molecules/Button/Button'
+import { useTranslation } from 'react-i18next'
+import { FC, ReactElement, useState } from 'react'
+import ProgressBar, { Step } from 'components/atoms/ProgressBar/ProgressBar'
+import { find, size } from 'lodash'
+
+interface FormDataProps {
+  title: string
+  helperText: string
+  modalContent: ReactElement | string
+}
+
+export interface FormProgressProps {
+  steps?: Step[]
+  formData?: FormDataProps[]
+  isModalOpen?: boolean
+  closeModal: () => void
+}
+
+const FormProgressModal: FC<FormProgressProps> = ({
+  steps,
+  formData,
```

Comment:
**[non-blocker]**

Not necessary, but a less error-prone data structure here would be to combine "formData" and "steps" props here.
Basically sth like


`
steps = [
    {
        label: 'Step1',
        title: 'Title1',
        helperText: 'Helper1',
        content: <div>Content1>
    },
    ...
]
`

This way it will be very clear what step matches what content + helps us avoid cases, where the size of "steps" and "formData" arrays don't match.

Reply:
Good Idea